### PR TITLE
Add doc type to initializers and methods.

### PR DESCRIPTION
**Pull request by kikettas**
### Description
Add the option to specify the doc type hint in the GVL initializers.

### How to test
Specify the docType in the GVL initializers and check that it's propagated to the Gini API SDK

### Merging
Automatic

### CODE REVIEW

File: GiniVision/Classes/Networking/AccountingDocumentService.swift [hunk 1]
```diff
@@ -6,19 +6,21 @@
 //
 
 import Foundation
-import Gini_iOS_SDK
+import Gini_iOS_SDK
```
Comment (a-szotyori) [inline GiniVision/Classes/Networking/AccountingDocumentService.swift]: Why is this import needed now?

Comment (kikettas) [reply GiniVision/Classes/Networking/AccountingDocumentService.swift]: It was already there. Not sure why the repo changed this file at that lien, myabe a line break or similar.
